release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -17,7 +17,7 @@ The release manager must:
 Whenever a new Kubernetes minor version is released, our kubernetes-csi CI jobs
 must be updated.
 
-[Our CI jobs](https://k8s-testgrid.appspot.com/sig-storage-csi-ci) have the
```

Comment:
This file is a git subtree so should be updated according to the steps here: https://github.com/kubernetes-csi/csi-release-tools/tree/master#sharing-and-updating